File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/CheckboxesVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

@Data
public class CheckboxesVo implements Serializable {
    private String menuName;
    private String menuId;
    private List<Long> defaultValue;
    private List<OperationsVo> operations;

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/CheckboxesVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable


@AllOpen
class CheckboxesVo : Serializable {
    var menuName: String? = null
    var menuId: String? = null
    var defaultValue: List<Long>? = null
    var operations: List<OperationsVo>? = null
}
